app/code/Magento/GoogleGtag/Block/Head.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\GoogleGtag\Block;
+
+use Magento\Framework\View\Element\Template;
+use Magento\Framework\View\Element\Template\Context;
+use Magento\GoogleGtag\Helper\GtagConfiguration;
+
+/**
+ * Google Ads Head block
+ *
+ * @api
+ */
+class Head extends Template
```

Comment:
Can this block be replaced by a view model?

See reference above.

Reply:
Moved this block to View model